modbus_rtu_block.py

```diff
@@ -56,6 +64,9 @@ class ModbusRTU(Retry, Block):
     retry = IntProperty(title='Number of Retries before Error',
                         default=10,
                         visible=False)
+    portconfig = ObjectProperty(PortConfig,
```

Comment:
@tyoungNIO, I'm slightly hesitant to merge this block since it's not backwards compatible, but until we move n.io 2.0 out of beta, I'm more comfortable doing things like this. 

Pretty picky, but I think think would look better as `port_config` instead of `portconfig`. A little more pythonic.